templates/debugging.md

```diff
@@ -0,0 +1,52 @@
+# Debugging the Provider
```

Comment:
Also, agai if this needs to be a template, why does it not need a header with page title etc as you have added elsewhere?

Reply:
`debugging.md` being in templates as a .md file and not .md.tmpl file just ensures it is copied as is. But it's a good suggestion to add the metadata/generated comment at the top for clarity, I'll do that now